entries/jQuery.htmlPrefilter.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.htmlPrefilter" return="String">
+  <title>jQuery.htmlPrefilter()</title>
+  <desc>Modify and filter HTML strings passed through <a href="/category/manipulation/">jQuery manipulation methods</a>.</desc>
+  <signature>
+    <added>1.12/2.2</added>
```

Comment:
I can't remember or find any method that has the double version. I'm fine with this line but I wonder if we have any other example for consistency.